docs/suspend-cluster.rst

```diff
@@ -0,0 +1,90 @@
+.. _suspend-cluster:
+
+=================
+Suspend a cluster
+=================
+
+This guide will provide a quick overview of how to suspend your cluster using
+the CrateDB Cloud Console. For the purposes of this guide, it is assumed that you have a cluster up and running and can access the Console. If not, please refer to the :ref:`tutorial on how to deploy a cluster for the first time
+<cloud-tutorials:cluster-deployment>`.
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+
+.. _what-is-suspendion:
+
+What is cluster suspension
+==========================
+
+Cluster suspension is a feature that enables you to pause the computation on
```

Comment:
Is "computation" the right word here? Seems unusual :)

Maybe something like "Cluster suspension is a feature that allows you to temporarily pause your cluster, retaining any data you have on it."